images/scripts/fedora-34.bootstrap

```diff
@@ -19,8 +19,8 @@
 
 set -ex
```

Comment:
... i.e. this script already has this, and was just missing `-u`. It somehow got lost in the f-33 bootstrap script.

Reply:
lol.  Good :)